ion/reader.go

```diff
@@ -90,13 +90,9 @@ type Reader interface {
 	// even if the Type is not NullType (for example, null.struct has type Struct).
 	IsNull() bool
 
-	// FieldName returns the field name associated with the current value. It returns
-	// nil if there is no current value or the current value has no field name.
-	FieldName() *string
-
-	// Annotations returns the set of annotations associated with the current value.
+	// Annotations returns the set of annotations associated with the current value as a list of SymbolTokens.
```

Comment:
```suggestion
	// Annotations returns the  annotations associated with the current value as a list of SymbolTokens.
```

not really a set and to avoid confusion I would remove that text fragment

Reply:
Removed